core/src/test/resources/HistoryServerExpectations/minDate_app_list_json_expectation.json

```diff
@@ -9,27 +24,25 @@
     "sparkUser" : "tgraves",
     "completed" : true,
     "appSparkVersion" : "3.0.0-SNAPSHOT",
+    "lastUpdatedEpoch" : 0,
     "endTimeEpoch" : 1578764802615,
-    "startTimeEpoch" : 1578764662851,
-    "lastUpdatedEpoch" : 0
+    "startTimeEpoch" : 1578764662851
```

Comment:
why the output order is different here?

Reply:
This is auto-generated when I ran `HistoryServerSuite.main`. I now changed to preserve the ordering to minimize the changes.